README.md

```diff
@@ -132,3 +132,9 @@ Shows the roles defined in your DAO with all the relevant info: where they are d
 #### `aragon devchain`
 
 Starts a local ganache test chain for development. This is done automatically with `aragon run` also if you don't have one already running.
+
+### Note
+
+There is  `.ipfsignore` file got created. These are the files and/or sub-directories which you don't want added to the pack. 
```

Comment:
I would add this to the [`aragon apm publish`](https://github.com/aragon/aragon-cli/blob/5b4dbf37ef7666ed7ecada73db4d607de41489e7/README.md#aragon-apm-publish) section instead, and note that having an `.ipfsignore` file will override the `.gitignore` (much like [`.npmignore`](https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package)).

Also, I'm not sure what the "pack manifest" you're referring to means.